### PR TITLE
removed deprecated eventName parameter, Fix #437

### LINKED ISSUE #437: Address // TODO: x.23 comments

### PR DESCRIPTION
**Pull request by vach-chronicle**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/wire/Event.java [hunk 1]
```diff
@@ -68,15 +68,9 @@ default E eventTimeNow() {
...
-     *
-     * @param eventName name of the event
+     * Updates event with event time to now if required.
      */
-    // TODO: x.23 remove eventName parameter
-    default E updateEvent(final String eventName) {
-        if (this.eventId().length() == 0)
-            this.eventId(eventName);
-
+    default E updateEvent() {
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/wire/Event.java:73]: ah actually I don't think we should do this any more. We may undeprecate eventName

**Review by JerryShea — CHANGES_REQUESTED** [review]
to be discussed

File: src/main/java/net/openhft/chronicle/wire/Event.java (hunk 1 above)
Comment (vach-chronicle) [reply src/main/java/net/openhft/chronicle/wire/Event.java:73]: great I was feeling unease about this one )

**Comment by JerryShea** [comment]
Let's close this one without merging. I think we have to bump deprecation of eventId to a later release until we better understand customer usage

**Comment by vach-chronicle** [comment]
Ok no problem, I'll keep the local brunch just in case

**Comment by JerryShea** [comment]
Replaced by #463

**Pull request closed without merging**